components/playground/instance/tiflash.go

```diff
@@ -139,6 +139,7 @@ func (inst *TiFlashInstance) Start(ctx context.Context, version utils.Version) e
 		fmt.Sprintf("--path=%s", filepath.Join(inst.Dir, "data")),
 		fmt.Sprintf("--listen_host=%s", inst.Host),
 		fmt.Sprintf("--logger.log=%s", inst.LogFile()),
+		"--logger.level=debug",
```

Comment:
can we overwrite this config when running playground？

Reply:
Fixed in https://github.com/pingcap/tiup/pull/2346/commits/7ccb33d9f4dcb2acf312922ab99c989bdec45d0b